src/api/public/apidocs-new/paths/search_project_id.yaml

```diff
@@ -28,10 +28,6 @@ get:
             - `group/@role`: name of the role which a group could be related with this project.
           - More complex predicates are: `maintenance/maintains/@project`, `repository/@name`,
             `repository/path/@project`, `repository/releasetarget/@trigger`.
-
-        These functions can also be used with predicates: `boolean`, `contains`, `not`, `starts_with`, `ends_with`.
-
-        More than one predicate can be combined with the following operators: `eq`, `and`, `or`, `neq`, `gt`, `lt`, `gteq`, `lteq`.
```

Comment:
With the recent merges of the last pull requests, there are more text like this one that could be removed, from other endpoints.

Reply:
@eduardoj done :)